score/container/container.go

```diff
@@ -62,6 +63,37 @@ func containerResources(requireCPULimit bool, requireMemoryLimit bool) func(core
 	}
 }
 
+// containerResourceRequestsEqualLimits checks that no container is using the ":latest" tag
+func containerResourceRequestsEqualLimits(podTemplate corev1.PodTemplateSpec, typeMeta metav1.TypeMeta) (score scorecard.TestScore) {
+	pod := podTemplate.Spec
+
+	allContainers := pod.InitContainers
+	allContainers = append(allContainers, pod.Containers...)
+
+	resourcesDoNotMatch := false
+
+	for _, container := range allContainers {
+		requests := &container.Resources.Requests
+		limits := &container.Resources.Limits
+		if !requests.Cpu().Equal(*limits.Cpu()) {
+			score.AddComment(container.Name, "CPU requests do not match limits", "Having equal requests and limits is recommended to avoid resource DDOS of the node during spikes. Set Set resources.requests.cpu == Set resources.limits.cpu")
```

Comment:
Did you mean this?

```suggestion
			score.AddComment(container.Name, "CPU requests does not match limits", "Having equal requests and limits is recommended to avoid resource DDOS of the node during spikes. Set resources.requests.cpu == resources.limits.cpu")
```